shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2644,31 +2659,55 @@ \subsection{Reminder: Stakepool Registration}
 \subsection{Epoch Rewards}
 \label{epoch-rewards}
 
-There will be four sources of rewards for an epoch: \emph{transaction
-  fees}, non-refundable parts of \emph{deposits}, \emph{monetary
-  expansion}, and rewards from the \emph{previous} epoch.
+There will be three sources of rewards for an epoch: \emph{transaction
+  fees}, non-refundable parts of \emph{deposits}, and \emph{monetary
+  expansion}.
 
 \subsubsection{Transaction Fees}
 \label{transaction-fees-1}
 
 All transaction fees from all transactions from all blocks created
-during the epoch will be used as rewards.
+during the epoch will be added to the rewards pot of that epoch.
 
 \subsubsection{Deposits}
 \label{deposits-1}
 
 As explained in \cref{fees-deposits}, a part of the deposits for
 certificates and UTxO entries are non-refundable, and contribute to
-the rewards pool instead.
+the rewards pot instead.
 
 \subsubsection{Monetary Expansion}
 \label{monetary-expansion}
 
-Let \(T\) be the total amount of ADA in existence during a specific
-epoch, and let \(T_\infty\) be the maximal possible amount of ADA in the
-future. At this moment, \(T=31,000,000,000\) and
-\(T_\infty=45,000,000,000\). Then the amount of
-\(\rho\cdot(T_\infty - T)\) ADA will be newly created.
+Every epoch, the total amount of ada in circulation \(T\) will be increased by
+adding ada to the rewards pot. To ensure the amount of ada never exceeds a
+finite, specified limit \(T_\infty\), the increase will reduce exponentially.
+Also, the increase will depend on the number of blocks that have been produced
+during the epoch. Specifically, each epoch, the contribution from monetary
+expansion to the rewards pot is given by
+\[\min(\eta, 1) \rho \left(T_\infty - T\right)\,,\]
+where
+\begin{itemize}
+\item[\(\eta\)] is the ratio between the number of blocks that have been produced
+  during the epoch, and the expectation value\footnote{Note that in Ouroboros
+    Classic, this is just the number of slots in an epoch, but for instance in
+    Praos, where we do not always have one leader per slot, we only have a
+    notion of how many blocks we expect per epoch \emph{on average}.} of blocks
+  that should have been produced during the epoch under ideal conditions (i.e.,
+  no forks, no missed blocks),
+\item[\(\rho\)] is the monetary expansion parameter,
+\item[\(T_\infty\)] is the maximal amount of ada to ever be in circulation
+  (i.e., \(45\cdot10^{9}\) ada), and
+\item[T] is the amount of ada in circulation at the beginning of the epoch for
+  which we want to calculate the rewards pot.
+\end{itemize}
+
+The dependece on \(\eta\) incentivises cooperative behaviour, and in particular
+discourages the pools sabotaging each others blocks. Note that \(\eta\) can
+exceed \(1\) when there are more blocks produced in an epoch than would be
+expected on average. But the product \(\min(\eta, 1)\rho\) is always bounded by
+\(1\), which is necessary to ensure we never exceed \(T_\infty\) ada in
+circulation.
```

Comment:
I think we should mention how \eta is actually calculated (provided it is _not_ simply the number of slots per epoch).